src/sentry/static/sentry/app/views/groupEvents.jsx

```diff
@@ -82,9 +82,11 @@ const GroupEvents = React.createClass({
           pageLinks: jqXHR.getResponseHeader('Link')
         });
       },
-      error: (error) => {
+      error: (err) => {
+        let error = err.responseJSON || true;
+        error = error.detail || true;
```

Comment:
you probably thing that because undefined.foo does throw - so you can't do foo.bar.baz || true, you have to do it in two steps (like this)